### ISSUE TITLE
Add Native App card back to the home page

### ISSUE THREAD

**Issue by kimLL21** [MNIT-DOT-a11y/mnit-dot-a11y.github.io#3]
Add the Native App card back to the home page. Place it to the right of the Web accessibility checklist card. The Native App card should be sized like the Web accessibility checklist card.
